.env.example

```diff
@@ -19,6 +19,9 @@ VUE_APP_NAMEX_API_URL="https://namex-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_NAMEX_API_VERSION="/api/v1"
 VUE_APP_STATUS_API_URL="https://status-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_STATUS_API_VERSION="/api/v1"
+VUE_APP_REGISTRIES_SEARCH_API_URL="https://bcregistry-dev.apigee.net/registry-search"
+VUE_APP_REGISTRIES_SEARCH_API_VERSION="/api/v1"
+VUE_APP_REGISTRIES_SEARCH_API_KEY=
```

Comment:
Looks good. Please also update ~/devops/vaults.env (which is used to populate the env keys from the 1Password vaults).

Reply:
I see

Reply:
I did